sdk/metricsadvisor/azure-ai-metricsadvisor/src/samples/java/com/azure/ai/metricsadvisor/administration/AnomalyDetectionConfigurationAsyncSample.java

```diff
@@ -168,17 +168,29 @@ private static void printDetectionConfiguration(AnomalyDetectionConfiguration de
                 .getSuppressCondition().getMinRatio());
 
         System.out.printf("- Hard Threshold Condition:%n");
-        System.out.printf(" - Lower bound: %s%n",
-            wholeSeriesDetectionCondition.getHardThresholdCondition()
-                .getLowerBound());
-        System.out.printf(" - Upper bound: %s%n",
-            wholeSeriesDetectionCondition.getHardThresholdCondition()
-                .getUpperBound());
-        System.out.printf(" - Suppress conditions: minimum number: %s; minimum ratio: %s%n",
-            wholeSeriesDetectionCondition.getHardThresholdCondition()
-                .getSuppressCondition().getMinNumber(),
-            wholeSeriesDetectionCondition.getHardThresholdCondition()
-                .getSuppressCondition().getMinRatio());
+        if ( wholeSeriesDetectionCondition.getHardThresholdCondition() != null ) {
```

Comment:
@anuchandy Seems like we are setting the hard threshold in the example. So could this be a bug that it is returned as null?

Reply:
Yes,  the return value is null after executing the [deletemetricandemomalydetectionconfiguration] process.

Reply:
Yes, the return value is null after executing the [deletemetricandemomalydetectionconfiguration] process.

Reply:
@anuchandy what is your idea?

Reply:
The sample enumerate all configs and print each of them. Though all conditions exist for the config that the sample creates, some conditions may not be set for other existing configs. Since the sample lists all configs, it needs to be prepared for handling null for such optional fields. I've added necessary checks in this [pr](https://github.com/Azure/azure-sdk-for-java/pull/22691).